Source/WebCore/platform/graphics/gstreamer/GStreamerCommon.cpp

```diff
@@ -340,7 +340,7 @@ Vector<String> extractGStreamerOptionsFromCommandLine()
         return { };
 
     Vector<String> options;
-    auto optionsString = String::fromUTF8(std::span(contents.get(), length));
+    auto optionsString = String::fromUTF8(unsafeMakeSpan(contents.get(), length));
```

Comment:
Probably it would be a good idea to add this to `GSpanExtras.h`:

```cpp
GMallocSpan<char> gFileGetContents(const char* path, GUniqueOutPtr<GError>&);
```

and then use it here and in the few other places where we have calls to `g_file_get_contents()`.

Reply:
I've added the helper, but the only other place where we could use it is ChassisType.cpp, and I don't think it actually helps there.